base_classes/NXwaveplate.nxdl.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
All existing base classes using double quotes here

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.